packages/data-point/lib/entity-types/entity-hash/resolve.test.js

```diff
@@ -86,7 +86,7 @@ describe('entity.hash.addKeys', () => {
       })
     })
   })
-  test('do nothing if addKeys is empty', () => {
+  test('do nothing if addKeys is empty | in hash collection c.2', () => {
```

Comment:
Can we add `It should` to the beginning of this description? I don't think we need the stuff from the `|` on.